docs/api-docs/apps/tutorials/sample-app-next/05-step-3-integrate-api.md

```diff
@@ -0,0 +1,920 @@
+
+2. At the top of the file, replace the existing import with the following:
+
+```js
+import { Box, Flex, Panel, Text } from '@bigcommerce/big-design';
+import { useEffect } from 'react';
+import Header from '../components/header';
+import { useSession } from '../context/session';
+import { useProducts } from '../lib/hooks';
+```
+
+3. Update the `Index` functional component.
+
+```js
+const Index = ({ context }: { context: string }) => {
+    // Destructure summary from useProducts
+    const { summary } = useProducts();
+    // Destructure setContext from useSession
+    const { setContext } = useSession();
+
+    useEffect(() => {
+        if (context) setContext(context);
+    }, [context, setContext]);
+
+    return (
+        <Panel header="Homepage">
+            {summary &&
+                <Flex>
+                    <Box marginRight="xLarge">
+                        <Text>Inventory Count</Text>
+                        <Text>{summary.inventory_count}</Text>
+                    </Box>
+                    <Box marginRight="xLarge">
+                        <Text>Variant Count</Text>
+                        <Text>{summary.variant_count}</Text>
+                    </Box>
+                    <Box>
+                        <Text>Primary Category</Text>
+                        <Text>{summary.primary_category_name}</Text>
+                    </Box>
+                </Flex>
+            }
+        </Panel>
+    );
+};
+
+export const getServerSideProps = async ({ query }) => ({
+    props: { context: query?.context ?? '' }
+});
+
+export default Index;
+```
+
+[View code in GitHub](https://github.com/bigcommerce/sample-app-nodejs/blob/step-3-add-database/pages/index.tsx)
+
+Once `summary` is available, it will create the `Flex` component with three `Box` components inside of it. `inventory_count`, `variant_count`, and `primary_category_name` will be populated with data returned from calling the `/catalog/summary` endpoint we added in [Add the Products endpoint](#add-the-products-endpoint). 
+
+For the complete list of properties returned by the `/catalog/summary` endpoint, see [Get a Catalog Summary](https://developer.bigcommerce.com/api-reference/store-management/catalog/summary/getcatalogsummary). 
+
+## Update the user interface
+
+1. In the root of the pages folder, open the `_app.tsx` file.
+
+2. Import the `Box` and `Header` components.
+
+```js
+import { Box, GlobalStyles } from '@bigcommerce/big-design';
+import Header from '../components/header';
+```
+
+3. Import `SessionProvider` from Context.
+
+```js
+import SessionProvider from '../context/session';
+```
+
+Your updated import statements should resemble the following:
+
+```js
+import { Box, GlobalStyles } from '@bigcommerce/big-design';
+import type { AppProps } from 'next/app';
+import Header from '../components/header';
+import SessionProvider from '../context/session';
+```
+
+4. Wrap `<Component {...pageProps} />` with the `SessionProvider`.
```

Comment:
🍹 we might want to expand this just a tad so devs have a better understanding of what we're doing and the significance of doing so... something like, "In order for React Context to properly propagate, we need to wrap `<Component {...pageProps} />` with the Context `SessionProvider`. This will ensure each page has access to the React Context."